### ISSUE TITLE
PARTICLE_WEIGHT_FACTOR does not appear to have an effect

### ISSUE THREAD

**Issue by btrettel** [firemodels/fds#6134]
For the wake drag reduction case I am making a verification test for, I've tried varying the actual number of particles while keeping the number of computational particles low by increasing the PARTICLE_WEIGHT_FACTOR. This option appears to have no effect. The problem may be limited to the wake drag reduction case, particularly if wake drag reduction has no effect. Either way, there does not appear to be any verification cases for the PARTICLE_WEIGHT_FACTOR. I am not entirely familiar with the option, so I thought I'd file this issue.

See below for the FDS file I am working with. Increasing the number of computational particles such that wake drag reduction has an effect seems to be too computationally expensive. Consequently, I have been testing 1000 computational particles with a PARTICLE_WEIGHT_FACTOR of either 1 or 10000. The terminal velocity for this case is -22.2860857248849 m/s in both cases.

[wake_drag_reduction_periodic.txt](https://github.com/firemodels/fds/files/1795209/wake_drag_reduction_periodic.txt)

**Comment by sbenkorichi** [comment]
You can simply upload your input file as .txt by modifying it, so github can accept it, instead of using google drive.

**Comment by btrettel** [comment]
Thanks, I've fixed the file.

**Comment by Er9y714** [comment]
I ran the file and tried to see smv output but smv fails to start, gives an (windows) error and closes. Or smv gives error when I try to load particles.
I tried with a smaller number of particles but the result is the same. 

This problem aside, are you sure that `FREEZE_VELOCITY=.TRUE.` is not affecting your calculation?

**Comment by Er9y714** [comment]
Update; the smv error sources  from `EB_PART_FILE`. When I disable it (default), smv can load and display particles.

**Comment by btrettel** [comment]
EB_PART_FILE is a FDS feature used only for debugging. It adds more precision to the particle locations and other tracked particle quantities. It is not compatible with Smokeview, and doesn't seem to be documented at the moment, so I'll take case of that next week. Sorry about this.

The FREEZE_VELOCITY option only affects the gas phase solver, not the drag on the particles. It's used for testing purposes so that the gas phase velocity stays constant.

**Comment by btrettel** [comment]
As far as I can tell, ``PARTICLE_WEIGHT_FACTOR`` is factored into the wake drag reduction subroutine. I followed the subroutine ``WAKE_REDUCTION`` in ``part.f90`` back to line 2591, which factors in ``LP%PWT``, the particle weight factor. This makes sense as the particle weight factor seems to work in general, just not in this case.

I got side tracked for a while trying to figure out why the units of line 2591 seemed to not match, before I remembered that the R in ``RVC`` stands for reciprocal. So the units work.

At this point I am going to make some verification cases without wake drag reduction to see what sort of effect ``PARTICLE_WEIGHT_FACTOR`` has in general. I suspect it will work as advertised, but we should have these test cases either way.
